src/main/java/org/assertj/core/api/recursive/comparison/FieldLocation.java

```diff
@@ -32,6 +30,7 @@ public int compareTo(final FieldLocation other) {
 
   @Override
   public boolean equals(final Object other) {
```

Comment:
I would introduce a unit test for this `equals`, like the one for `Tuple`. See:

https://github.com/joel-costigliola/assertj-core/blob/45eb16da4a2290a0d3f9c77f7806372507705af6/src/test/java/org/assertj/core/api/Tuple_Test.java#L85-L90

Reply:
Okay, Let me implement it